developer/tutorial/plugin-layouts-3.md

```diff
@@ -27,40 +29,77 @@ class CoreLayoutBeesknees extends Component {
     structure: PropTypes.object
   }
 
+  getComponent(name) {
```

Comment:
I don't understand the purpose of this code that just wraps `getComponent`?

Reply:
This is just because getComponent throws if the component is not registered. The example needs a more forgiving approach because sometimes the requested component will no exist (in case the param is referring to a Blaze template.)

But as I said earlier, I don't like this code. It's ugly and it would not be necessary, if the `AboutUs` page would be implemented as Blaze template as well. This is a bit retrominded, but the code is clearer for our audience.

Reply:
Ok, after we talked this morning, I would leave as is for now..